index.html

```diff
@@ -0,0 +1,189 @@
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <h2 class="center">About me</h2>
+      <p class="center">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="cont">
+        <button class="see center-button">Get My Resume</button>
+      </div>
+    </section>
+    <section>
```

Comment:
- [ ] I would suggest you please use the background image to stick to the design. Also use the `color: #42526E;` for p tag to match the design.
### Your design
![his](https://user-images.githubusercontent.com/81727988/219549091-0cc567de-c532-456d-9c53-211962c16113.png)


### Figma design

![figma](https://user-images.githubusercontent.com/81727988/219549107-21376ace-2eec-4c39-a548-ee2c969720eb.png)

Reply:
completed